src/screens/NetworkSettings/CustomNetworkSettingsScreen.js

```diff
@@ -0,0 +1,222 @@
+import React, { useEffect, useState } from 'react';
+import { View, Text, TextInput, Button, StyleSheet } from 'react-native';
+import { useDispatch, useSelector } from 'react-redux';
+import { t } from 'ttag';
+import { networkSettingsUpdate, networkSettingsUpdateReady } from '../../actions';
+import FeedbackModal from '../../components/FeedbackModal';
+import HathorHeader from '../../components/HathorHeader';
+import NewHathorButton from '../../components/NewHathorButton';
+import SimpleInput from '../../components/SimpleInput';
+import { NETWORKSETTINGS_STATUS } from '../../constants';
+import errorIcon from '../../assets/images/icErrorBig.png';
+import Spinner from '../../components/Spinner';
+
+const customNetworkSettingsTitleText = t`Custom Network Settings`.toUpperCase();
+const warningText = t`Any change to the network settings cannot be validated by Hathor. Only change if you know what you are doing.`;
+const feedbackLoadingText = t`Updating custom network settings...`;
+const feedbackFailedText = t`There was an error while customizing network settings. Please try again later.`;
+
+/**
+ * Check if the network settings status is failed.
+ * @param {object} networkSettingsStatus - status from redux store
+ * @returns {boolean} - true if the status is failed, false otherwise
+ */
+const hasFailed = (networkSettingsStatus) => {
+  return networkSettingsStatus === NETWORKSETTINGS_STATUS.FAILED;
+};
+
+/**
+ * Check if the network settings status is loading.
+ * @param {object} networkSettingsStatus - status from redux store
+ * @returns {boolean} - true if the status is loading, false otherwise
+ */
+const isLoading = (networkSettingsStatus) => {
+  return networkSettingsStatus === NETWORKSETTINGS_STATUS.LOADING;
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  content: {
+    flex: 1,
+    padding: 16,
+    paddingBottom: 48,
+  },
+  feedbackModalIcon: {
+    height: 105,
+    width: 105
+  },
+  warningContainer: {
+    borderRadius: 8,
+    backgroundColor: 'hsl(47, 100%, 62%)', // warning yellow
+    marginBottom: 16,
+  },
```

Comment:
I don't think I like this warning container UI.

Can we try other possibilities? 

Maybe a lighter yellow. Maybe with a white text.

Maybe similar to this warning alert: https://getbootstrap.com/docs/5.3/components/alerts/

I really like bootstrap colors, and we use it in the desktop

Reply:
- Refactor warning style: 14129aa12569516cfa24848cfa82cb24ecc349e0